modules/sdk/node-build/cssLoad/writeCSSLoaderJavaScriptModules.mjs

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import getFlatName from "../util/getFlatName.mjs";
+
+export default async function writeCSSLoaderJavaScriptModules(projectExports, projectWebContextPath) {
+	return Promise.all( 
+		projectExports
+			.filter(moduleName => moduleName.endsWith('.css'))
+			.map(moduleName => writeCSSLoaderJavaScriptModule(projectWebContextPath, moduleName))
+	);
+}
+
+async function writeCSSLoaderJavaScriptModule(webContextPath, moduleName) {
+	const flatModuleName = getFlatName(moduleName);
+
+	const cssLoaderPath = 
+		`./build/node/packageRunBuild/resources/__liferay__/exports/${flatModuleName}.js`;
```

Comment:
nit: might be worth having a file with constants for things like `/build/node/packageRunBuild/resources/__liferay__/exports/`

Reply:
done